immunopepper/immunopepper_filter.py

```diff
@@ -84,4 +96,131 @@ def immunopepper_filter(arg):
     logging.info(">>>>>>>>> filter: Finish\n")
 
 
+def get_start_pos_for_kmer_unfiltered(meta_file_path,junction_kmer_tsv_path,output_file_path,is_compressed=True):
+    """
+    Given kmer_junction_file, return the exact dna positions that output the given kmer.
+    Parameters
+    ----------
+    meta_file_path: _metadata.tsv(.gz) file outputed by build mode.
+    junction_kmer_tsv_path: _junction_kmer.txt by build mode.
+    output_file_path: the specified output file path.
+    is_compressed: Bool. If output compressed file.
+
+    Return:
+    output_file_path: str. return the true output file name (in some cases will add .gz)
+    """
+    meta_df = pd.read_csv(meta_file_path,sep='\t',usecols=[0,3,4,11,13])
```

Comment:
I would suggest to rather use the column names in the `read_csv` by using `names=['x', 'y', 'z']`, rather than `usecols=[1, 2, 4]`.

Reply:
Will take care of this in a later PR